Source/Core/VRTheWorldTerrainProvider.js

```diff
@@ -268,7 +268,7 @@ define([
             request : request
         });
         var promise = resource.fetchImage({
-            flipY : false
+            preferImageBitmap: true
```

Comment:
There are a few places including here that don't pass in `preferBlob: true`. Should they?

Reply:
Especially curious about `ImageryProvider` not using `preferBlob` since `preferBlob` is helpful for `ImageBitmap`.

Reply:
I'm actually not sure when in general we want to set `preferBlob: true`. @mramato do you have any insight here?

In terms of using ImageBitmap, `preferImageBitmap` forces it to fetch it as a blob, otherwise decoding is locked to the main thread.

Reply:
`preferBlob` is something you don't want to set unless you are working around limitations inherent in using an Image element.

Some of these include:

1. Sending headers with the request, for example an Authorization header with an access token in it for secured urls, can't be accomplished with an Image element.
2. Getting the size/length of the resulting data `DiscardMissingTileImagePolicy` does this.

It's possible I'm forgetting something, but I think those are the only 2.  If `preferImageBitmap` implies preferBlob is true, that's fine and the end result is transparent to the user.

Reply:
So, if you set `preferBlob: true`, it still gives you back an `Image`. It just stores the `blob` to it as well so you can do things like read the bytes or size etc. right?

The `image.blob = blob` is not set when using ImageBitmap, so I'm surprised nothing has broken there. Also, it looks like if an ImageryProvider has a discard policy, it's going to load every tile with its blob?

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/ImageryProvider.js#L347-L349

Reply:
It looks like this is necessary because when you first create the DiscardMissingTileImagePolicy, it does one request, to check what the missing tile looks like? And then that's why we need `preferBlob: true` on every subsequent request, to check if what we got is a missing tile or a real tile, if I'm reading this right.

Reply:
Ok, so looks like all we need to do here is make sure when you set `preferBlob: true` you can access the blob, regardless of whether it's loading using `Image` or `ImageBitmap`. But when you want to use `preferBlob` and when you want to use `preferImageBitmap` are unrelated.

Reply:
And part of that was my confusion because I didn't notice that `ImageBitmap` was always requesting a blob regardless of the `preferBlob` setting. So `preferBlob` is orthogonal and its main purpose is to just attach a blob to the `Image`/`ImageBitmap` to be used by `DiscardMissingTileImagePolicy`.

Reply:
Yes, but also to send headers.  If preferBlob is not set and there are headers with the request, we use a blob.